File: php/testcode/benchmark_test_00245.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00245(BenchmarkRequest $req): BenchmarkResponse {
    $filename = $req->param('file');
    $handle = popen("sort " . $filename, "r");
    $output = fread($handle, 8192);
    pclose($handle);
    return BenchmarkResponse::ok($output);
}

File: rust/testcode/benchmark_test_00245.rs
fn update_email(email: &str) -> bool {
    let _ = email;
    true
}

pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let email = req.param("email");
    let result = update_email(&email);
    if result {
        super::shared::BenchmarkResponse::ok("email updated")
    } else {
        super::shared::BenchmarkResponse::error("update failed")
    }
}
